src/twisted/protocols/policies.py

```diff
@@ -124,6 +124,9 @@ def connectionLost(self, reason):
         self.factory.unregisterProtocol(self)
         self.wrappedProtocol.connectionLost(reason)
 
+        # Breaking reference cycle between self and wrappedProtocol
```

Comment:
Use full for a sentence.
In this way, it is clear that the sentence has an end and that it was not left unfinished by accident.